_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,136 @@
+
+4. Select **Create monitor** or **Associate monitor**, as applicable, to add the monitor to your visualization, as shown in the following images.
+
+| Create monitor | Associate monitor |
+|:--------------:|:-----------------:|
+| ![Create monitor interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/save-new-monitor.png) | ![Associate monitor interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/save-associated-monitor.png) |
+
+### Viewing events
+
+Regardless of whether you choose to create a new monitor or detector or to associate an existing monitor or detector, events on your dashboard are triggered based on the criteria you set up. For a detailed view of the trigger criteria, go to the **View Events** page. You have two options for accessing the **View Events** menu: Select an event on the visualization itself or select **View Events** from the visualization’s context menu, as shown in the following image.
+
+![View events context menu interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/view-events-context-menu.png)
+##### Figure 6. _View events_ context menu
+
+As shown in the following image, the events flyout has two sections. The top section is the visualization displaying the same aggregate view of events and time range as the visualization on the dashboard. The events flyout can be refreshed to fetch any new data. The bottom section displays the specific event details. This section is arranged by plugin (Anomaly Detection or Alerting), then by the individual plugin resources within that plugin (anomaly detectors or alerting monitors). In this view, you can see which plugin resources are producing events and when those events were produced. An overall event count for each plugin resource is displayed next to the resource name. To display more information about a specific plugin resource or the event it has produced, select the resource name. This opens a new tab that displays the resource details about that plugin. A time range is also fixed at the bottom of the flyout to help visually correlate individual events.
+
+![View events flyout interface]({{site.url}}{{site.baseurl}}/assets/media/blog-images/2023-07-20-alert-anomaly-visual.md/events-flyout.png)
+##### Figure 7. Events flyout showing alerting and anomaly detectors details and a line chart visualization
+
+We're excited to announce that we're continuing to work on this feature! We're adding new components and making them common components so that you can use them in your own applications. We're also working to remove these limitations: the number of objects associated with visualizations is 10 and only line chart visualizations containing time-series data is supported. We'll have updates in the upcoming release, so stay tuned!
```

Comment:
```suggestion
As part of our continuing work on this feature, we're adding new components and making them common components so that you can use them in your own applications. We're also working to remove these limitations: 
- The number of objects associated with visualizations is 10.
- Only line chart visualizations containing time-series data are supported. 

We'll have updates in the upcoming release, so stay tuned!
```